package.json

```diff
@@ -52,6 +52,7 @@
     "@storybook/builder-webpack5": "^6.5.0-alpha.34",
     "@storybook/manager-webpack5": "^6.5.0-alpha.34",
     "@storybook/react": "^6.5.0-alpha.34",
+    "@testing-library/cypress": "^8.0.2",
```

Comment:
what's this library giving us? interested in why we haven't needed it until now

Reply:
It allows you to use the same queries you would be using with `@testing-library/react`. So for instance:

```js
const checkbox = cy.findByLabelText('Yes, sign me up');
```

instead of

```
const checkbox = cy
    .contains('label', 'Yes, sign me up')
    .invoke('attr', 'for')
    .then((id) => {
      cy.get('#' + id)
    })
```

If you associated a label with an input by just wrapping it, then you'd probably need to do something even more complicated.

Unlike `@testing-library/react` it doesn't give you the same nice assertion functions like `.toBeInTheDocument()` etc but I still think it's worth having, should encourage us to avoid testing implementation details, and make context switching between the jsdom and cypress tests a bit easier too hopefully

Reply:
brilliant, thanks for covering the key changes, sounds like a great addition to our testing stack